src/main/java/edu/cornell/kfs/coa/document/validation/impl/SubAccountGlobalRule.java

```diff
@@ -98,7 +111,7 @@ public boolean checkSubAccountDetails() {
 		SubAccountGlobal newSubAccountGlobal = (SubAccountGlobal) super.getNewBo();
 		List<SubAccountGlobalDetail> details = newSubAccountGlobal.getSubAccountGlobalDetails();
 		// check if there are any accounts
-		if (details.size() == 0) {
+		if (details.size() == 0 && newSubAccountSectionHasEmptyValues(newSubAccountGlobal)) {
 			putFieldError(KFSConstants.MAINTENANCE_ADD_PREFIX + CUKFSPropertyConstants.SUB_ACCOUNT_GLBL_CHANGE_DETAILS + "." + KFSPropertyConstants.ACCOUNT_NUMBER, CUKFSKeyConstants.ERROR_DOCUMENT_GLOBAL_SUS_ACCOUNT_NO_SUB_ACCOUNTS);
```

Comment:
Should we use an updated error message here to include there is no details and no new sub accounts

Reply:
I did update this error message's text already; see the CU-ApplicationResources.properties changeset for details. Let me know if you think the message needs further modifications.

Reply:
oh yeah, I noticed that when I got lower in the PR, I forgot to remove this comment.  I think iit is OK to leave as is.